documentation/config_file_puppetserver.markdown

```diff
@@ -73,6 +73,19 @@ The `puppetserver.conf` file contains settings for Puppet Server software. For a
 * The `profiler` settings configure profiling:
 
     * `enabled`: If this is set to `true`, Puppet Server enables profiling for the Puppet Ruby code. The default is `false`.
+    
+* The `product` settings configure update checking and analytics data collection:
```

Comment:
@gguillotte the actual file/file name that the setting is in doesn't actually matter for puppetserver. So long as the top level `product` key is somewhere in the directory, it will be picked up.

That said, I don't think we have any other files other than puppetserver.conf that we currently create by default, and IMO if we do want to separate out the `product` key then why not separate out `webserver` and `jruby-puppet` etc. etc. I think the arguments you're making for `product` could apply to the rest of those too.